components/StreamField/StreamField.module.scss

```diff
@@ -1,33 +1,33 @@
 .container {
-    @include media-query(large) {
-        margin: 0 calc(5rem + 80px);
-    }
-}
-
-.text {
-    line-height: 1.5;
+    &__topic-page {
```

Comment:
**Nitpick** - typically with BEM I'd use a `&--topic-page` here instead of the `__` as I feel this is more like a modifier class. The same goes for `__home-page` below.

Reply:
Oh, yes thanks!